.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: objective-c
 notifications:
   email: false
-osx_image: xcode7.2
+osx_image: xcode7.3
 before_install:
-  - brew update
-  - brew upgrade carthage
```

Comment:
I think we should keep this so that Tentacle automatically uses the latest Carthage release.